db/migrate/20150912205745_create_seed_trades.rb

```diff
@@ -0,0 +1,17 @@
+class CreateSeedTrades < ActiveRecord::Migration
+  def change
+    create_table :seed_trades do |t|
+      t.text       :message
+      t.text       :address
+      t.datetime   :requested_date
+      t.datetime   :accepted_date
```

Comment:
As per discussion on the earlier pull request, I think we want to change this to be replied date and what the reply was, rather than accepted date and declined date. I know Skud's original issue had these, but I don't think we'd thought it through at that stage.